admin-core/assets/src/dashboard-app/pages/welcome/Welcome.js

```diff
@@ -179,6 +179,95 @@ const Welcome = () => {
 								</div>
 							</div>
 						</section>
+						<section aria-labelledby="section-2-title">
+							<h2 className="sr-only" id="section-2-title">
+								Section title
+							</h2>
+							<div className="box-border rounded-md bg-white shadow-sm overflow-hidden transition hover:shadow-hover">
+								<div className="p-8 pr-10">
+									<h3 className="text-slate-800 text-xl font-medium pb-2">
+										{ __( 'Rate Us', 'ultimate-addons-for-gutenberg' ) }
+									</h3>
+									<p className="text-slate-500 text-sm pb-2.5 pr-12 flex items-center">
+										<span className="text-xl text-slate-800 flex mr-2">
+											{ [ 1, 2, 3, 4, 5 ].map( ( item, key ) => (
+												<svg
+													key={ key }
+													width="20"
+													height="20"
+													viewBox="0 0 20 20"
+													fill="none"
+													xmlns="http://www.w3.org/2000/svg"
+												>
+													<defs>
+														<linearGradient
+															id={ `half-color-gradient-${ key }` }
+															x1="0%"
+															x2="100%"
+															y1="0%"
+															y2="0%"
+														>
+															<stop
+																offset="50%"
+																style={ {
+																	stopColor: 'rgb(30, 41, 59)',
+																	stopOpacity: 1,
+																} }
+															/>
+															<stop
+																offset="50%"
+																style={ { stopColor: 'grey', stopOpacity: 1 } }
```

Comment:
Try Tailwind 500 here.
```suggestion
																style={ { stopColor: '#64748b', stopOpacity: 1 } }
```

Reply:
Okay...I'll do the changes.